### DISCUSSION TITLE
Canvas Display Texture?

### DISCUSSION THREAD

**Discussion by N8python** [enable3d/enable3d#140] [Help / Questions] [answered]
Is there any easy way to create a `CanvasTexture` for a label in `enable3d`? Or should I just use the default threejs way?

**Comment by yandeu** [comment, marked as answer]
Have a look at the **2D Elements (Experimental)** section on the [examples page](https://enable3d.io/examples.html).

**Reply by N8python** [reply]
Awesome - thanks!
